app/controllers/events/list.js

```diff
@@ -75,6 +75,7 @@ export default Controller.extend({
       })
         .then(() => {
           this.notify.success(this.get('l10n').t('Event has been deleted successfully.'));
+          this.send('refreshRoute');
```

Comment:
Where is this `refreshRoute` action defined? @kushthedude

Reply:
I think it was defined in route , Let me give a look will push commit if not defined @CosmicCoder96